### ISSUE TITLE
Reproducing NVIDIA winning solution

### ISSUE THREAD

**Issue by gaceladri** [coveooss/SIGIR-ecom-data-challenge#3]
Hi, I was trying to reproduce the [NVIDIA Merlin wining solution ](https://github.com/NVIDIA-Merlin/competitions/tree/main/SIGIR_eCommerce_Challenge_2021) but they have the test set that I don't know where I can download it. With the download link from the official challenge, the .zip only have the training set. Is it possible to download the test set also? Thanks!

**Comment by jacopotagliabue** [comment]
Hi @gaceladri ! Thanks for your interest - the test set has not been released: however we have some tips in the README that people used during the challenge to generate a local test set for development purposes (i.e. the dataset is big enough to permit that). Results should be very similar to the official test set, given the sliding window split we used in the challenge (i.e. train on x weeks, test on unseen data in weeks x+k). Thanks again for reaching out!
